client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -72,52 +61,39 @@ const DesignSelector: React.FunctionComponent = () => {
 			</div>
 			<div className="design-selector__design-grid">
 				<div className="design-selector__grid">
-					{ designs.featured.map( design => {
-						const isFocused = hoverDesign === design.slug || focusDesign === design.slug;
-						return (
-							<Spring
-								native
-								key={ design.slug }
-								from={ ZOOM_OFF }
-								to={ isFocused ? ZOOM_ON : ZOOM_OFF }
-							>
-								{ ( props: React.CSSProperties ) => (
-									<animated.button
-										style={ props }
-										onMouseEnter={ () => setHoverDesign( design.slug ) }
-										onMouseLeave={ () =>
-											setHoverDesign( s => ( s === design.slug ? undefined : s ) )
-										}
-										onFocus={ () => setFocusDesign( design.slug ) }
-										onBlur={ () => setFocusDesign( s => ( s === design.slug ? undefined : s ) ) }
-										className="design-selector__design-option"
-										onClick={ () => {
-											setSelectedDesign( design );
-											if ( isEnabled( 'gutenboarding/style-preview' ) ) {
-												push( makePath( Step.Style ) );
-											}
-										} }
-									>
-										<Spring
-											native
-											key={ design.slug }
-											from={ SHADOW_OFF }
-											to={ isFocused ? SHADOW_ON : SHADOW_OFF }
-										>
-											{ ( props2: React.CSSProperties ) => (
-												<animated.span style={ props2 } className="design-selector__image-frame">
-													<img alt={ design.title } src={ getDesignUrl( design ) } />
-												</animated.span>
-											) }
-										</Spring>
-										<span className="design-selector__option-overlay">
-											<span className="design-selector__option-name">{ design.title }</span>
-										</span>
-									</animated.button>
+					{ designs.featured.map( design => (
+						<button
+							onClick={ () => {
+								setSelectedDesign( design );
+								if ( isEnabled( 'gutenboarding/style-preview' ) ) {
+									push( makePath( Step.Style ) );
+								}
+							} }
+							key={ design.slug }
+							className="design-selector__design-option"
+						>
+							<span className="design-selector__image-frame">
+								{ // Temporarily show pre-generated screenshots until we can generate tall versions dynamically using mshots
+								// https://github.com/Automattic/mShots/issues/
+								// https://github.com/Automattic/wp-calypso/pull/40632
+								isEnabled( 'gutenboarding/mshot-preview-preload' ) ? (
+									<MShotsImage
+										className={ `design-selector__preview-image-${ design.slug }` }
+										alt={ design.title }
+										siteUrl={ getDesignUrl( design ) }
+										key={ design.slug }
+									/>
+								) : (
+									<img
+										src={ `/calypso/page-templates/design-screenshots/${ design.slug }_${ design.template }_${ design.theme }.jpg` }
```

Comment:
@simison Would something like this satisfy the requirements of #40632 ?

Reply:
I love this change. Once MShots is capable of longer screenshots, it will be rather easy to change this code back. Thanks!